src/coffee/import.coffee

```diff
@@ -43,10 +43,12 @@ class Import
     options.importFormat = options.importFormat || "csv"
     options.csvDelimiter = options.csvDelimiter || ","
     options.encoding = options.encoding || "utf-8"
+    options.mergeCategoryOrderHints = !!options.mergeCategoryOrderHints
```

Comment:
I know, `Boolean` works practically the same way right? Only it's a bit more expressive which I like, it's very easy to read over '!!'.